fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -209,7 +209,7 @@ WD=`defaultopt $WD ${RegOutput}.wdir`
 dof=`defaultopt $dof 6`
 GlobalScripts=${HCPPIPEDIR_Global}
 TopupConfig=`defaultopt $TopupConfig ${HCPPIPEDIR_Config}/b02b0.cnf`
-Preregister=${Preregister:-epi_reg}
+PreregisterTool=${PreregisterTool:-epi_reg}
```

Comment:
Maybe this should also use `defaultopt` for consistency?

I'm not really sure which style I prefer of these 2, and the `defaultopt` implementation is not great (can't handle quoted spaces even by adding quotes, because it abuses unquoted bash argument parsing rather than using a conditional).

As a side note, `newopts.shlib` approaches this in the other direction, by setting the default values before checking what options are specified, so specifying the empty string doesn't result in using the default.